packages/datastore/datastore-codegen/tests/__snapshots__/Cli.test.ts.snap

```diff
@@ -0,0 +1,101 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`generate 1`] = `
+"export interface Note {
+    _id: string;
+    title: string;
+    description: string;
+    comments: string[]
+    _version?: number;
+    _deleted?: boolean;
+}
+
+export type NoteCreate = Omit<Note, \\"_id\\">;
+export type NoteChange =  Pick<Note, \\"_id\\"> & Partial<NoteCreate>;
+
+export interface Comment {
+    _id: string;
+    title: string;
+    description: string;
+    noteComment: string
+    _version?: number;
+    _deleted?: boolean;
+}
+
+export type CommentCreate = Omit<Comment, \\"_id\\">;
+export type CommentChange =  Pick<Comment, \\"_id\\"> & Partial<CommentCreate>;
+"
+`;
+
+exports[`generate 2`] = `
+"{
+  \\"Note\\": {
+    \\"name\\": \\"Note\\",
+    \\"version\\": 1,
+    \\"type\\": \\"object\\",
+    \\"primaryKey\\": \\"_id\\",
+    \\"properties\\": {
+      \\"_id\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"_id\\",
+        \\"isRequired\\": true,
+        \\"primary\\": true
+      },
+      \\"title\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"title\\",
+        \\"isRequired\\": true
+      },
+      \\"description\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"description\\",
+        \\"isRequired\\": true
+      },
+      \\"comments\\": {
+        \\"type\\": \\"string[]\\",
+        \\"key\\": \\"comments\\",
+        \\"isRequired\\": true
+      }
+    }
+  },
+  \\"Comment\\": {
+    \\"name\\": \\"Comment\\",
+    \\"version\\": 1,
+    \\"type\\": \\"object\\",
+    \\"primaryKey\\": \\"_id\\",
+    \\"properties\\": {
+      \\"_id\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"_id\\",
+        \\"isRequired\\": true,
+        \\"primary\\": true
+      },
+      \\"title\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"title\\",
+        \\"isRequired\\": true
+      },
+      \\"description\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"description\\",
+        \\"isRequired\\": true
+      },
+      \\"noteComment\\": {
+        \\"type\\": \\"string\\",
+        \\"key\\": \\"noteComment\\",
+        \\"isRequired\\": true
+      }
+    }
+  }
+}"
+`;
+
+exports[`generate 3`] = `
+"import { GeneratedModelSchema } from \\"offix-datastore\\";
+import jsonSchema from \\"./schema.json\\";
+
+export const schema = jsonSchema as GeneratedModelSchema;
+
+export * from \\"./types\\";
```

Comment:
types?

Reply:
Yes. This line(99) in generated/index.ts exports all types from generated/types

Reply:
This will not work IMHO. Schema is exported (but not going to get used anyway).
And types are precreated (but you cannot override schema).
We need to cover how to do overrides for that stuff. 

We need discussion about this. Maybe we should simply use GraphQL schema indexes and never allow to override schema.